src/actions.test.js

```diff
@@ -83,4 +85,17 @@ describe('actions', () => {
       });
     });
   });
+
+  describe('loadRestaurant', () => {
+    beforeEach(() => {
+      store = mockStore({});
+    });
+    it('runs setRestaurant', async () => {
+      await store.dispatch(loadRestaurant({ restaurantId: 1 }));
+
+      const actions = store.getActions();
+
+      expect(actions[0]).toEqual(setRestaurant({}));
```

Comment:
```suggestion
      expect(actions[0]).toEqual({ type: 'setRestaurant', payload: { restaurant } });
```
액션이 어떤 모양인지 표현해주면 더 좋을 것 같아요. 테스트는 설명서니까요ㅎㅎ